packages/fx-core/src/component/generator/apiSpec/helper.ts

```diff
@@ -801,7 +801,11 @@ async function updatePromptForCustomApi(
   language: string,
   chatFolder: string
 ): Promise<void> {
-  if (language === ProgrammingLanguage.JS || language === ProgrammingLanguage.TS) {
+  if (
+    language === ProgrammingLanguage.JS ||
+    language === ProgrammingLanguage.TS ||
+    language === ProgrammingLanguage.PY
+  ) {
```

Comment:
It is better to use meaningful condition like 
```javascript
// maintain an array to support ApiSpec feature
SupportApiLanguages = [
ProgrammingLanguage.JS,
ProgrammingLanguage.TS
ProgrammingLanguage.PY
]; 
// use condition like
SupportApiLanguages.includes(langueage)
```

Reply:
Updated.